src/main/java/com/hibernate/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,47 @@
+package com.hibernate.cinema.model;
+
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "chopping_carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(cascade = CascadeType.REMOVE)
+    private List<Ticket> tickets;
+    @OneToOne(cascade = CascadeType.REMOVE)
```

Comment:
Are you sure you need this cascade type here? I think this will remove the user if you remove the shopping cart

Reply:
But what can users do without a shopping cart?

Reply:
True. But maybe we want to keep the users' old shopping carts and purchased tickets to analyze the data later. I remember Bodia said that deleting stuff from the DB is not always a good idea.